src/tools/material.js

```diff
@@ -27,6 +26,7 @@ function scaleOneLatticeVector(material, key = "a", factor = 1.0) {
  * @param material {Material}
  */
 function scaleLatticeToMakeNonPeriodic(material) {
+    material.toCartesian();
```

Comment:
We should add a couple of tests to assert situation arising in this task doesn't happen in the future:

1. Repeatedly apply `getBasisConfigTranslatedToCenter` and assert nothing happens second time
2. Same for scaleLatticeToMakeNonPeriodic
3. Repeatedly apply `scaleLatticeToMakeNonPeriodic(getBasisConfigTranslatedToCenter ())` and assert nothing happens the second time